services/ni/plausible/03-config-map.yaml

```diff
@@ -10,4 +10,48 @@ data:
   SMTP_HOST_SSL_ENABLED: "true"
   SMTP_HOST_PORT: "465"
   BASE_URL: https://plausible.niaefeup.pt
-  CLICKHOUSE_DATABASE_URL: http://plausible-events-db:8123/plausible_events_db
\ No newline at end of file
+  CLICKHOUSE_DATABASE_URL: http://plausible-events-db:8123/plausible_events_db
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: clickhouse-config
+  namespace: plausible-ni
+data:
+  docker_related_config.xml: |
+    <clickhouse>
+        <!-- Listen wildcard address to allow accepting connections from other containers and host network. -->
+        <listen_host>::</listen_host>
+        <listen_host>0.0.0.0</listen_host>
+        <listen_try>1</listen_try>
+
+        <!--
+        <logger>
+            <console>1</console>
+        </logger>
+        -->
+    </clickhouse>
+  z_log_disable.xml: |
+    <?xml version="1.0"?>
+    <clickhouse>
+        <asynchronous_metric_log remove="1"/>
+        <backup_log remove="1"/>
+        <metric_log remove="1"/>
+        <part_log remove="1"/>
+        <session_log remove="1"/>
+        <text_log remove="1" />
+        <trace_log remove="1"/>
+        <crash_log remove="1"/>
+        <opentelemetry_span_log remove="1"/>
+        <zookeeper_log remove="1"/>
```

Comment:
We don't seem to be using these services, so maybe not needed to disable these -- however it also should not hurt

Reply:
I've disabled pretty much everything that's not essential for plausible operation